### PR TITLE
Removing unused exports across the codebase

### PR DESCRIPTION
**Pull request by emreboga**
I've used `ts-unused-exports` to detect unused exports and removed the ones that has references in the code.

### CODE REVIEW

File: wormhole-connect/package.json [hunk 1]
```diff
@@ -88,7 +88,8 @@
     "test": "vitest run tests/ci",
     "test:dev": "vitest",
     "checksdn": "npx tsx scripts/ofac/checkSdnListForUpdates.ts",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "unused:exports": "ts-unused-exports ./tsconfig.json --ignoreLocallyUsed"
```
Comment (kev1n-peters) [inline wormhole-connect/package.json:89]: Ohh, I like it

**Comment by artursapek** [comment]
tiger-tiger-woods

File: wormhole-connect/src/utils/confirmBeforeLeaving.ts [hunk 2]
```diff
@@ -1,24 +0,0 @@
-import { useEffect } from 'react';
-
-const unloadHandler = (e: Event) => {
-  e = e ?? window.event;
-  if (e) e.returnValue = true;
-  return true;
-};
-
-const useConfirmBeforeLeaving = (shouldConfirm: boolean) => {
```
Comment (kev1n-peters) [inline wormhole-connect/src/utils/confirmBeforeLeaving.ts:9]: This seems like a useful hook. Should we keep it and use it when a transfer is being performed? cc @artursapek

Comment (emreboga) [reply wormhole-connect/src/utils/confirmBeforeLeaving.ts:9]: I've added it back under hooks.

**Review by kev1n-peters — APPROVED** [review]

**Pull request merged**